WordPress/src/test/java/org/wordpress/android/ui/stories/SaveStoryGutenbergBlockUseCaseTest.kt

```diff
@@ -0,0 +1,288 @@
+                mediaFile
+        )
+
+        // Then
+        Assertions.assertThat(onePost.content).isEqualTo(blockWithNonEmptyMediaFilesWithOneRemoteId)
+    }
+
+    @Test
+    fun `slides are saved locally to storiedPrefs`() {
+        // Given
+        val frames = ArrayList<StoryFrameItem>()
+        frames.add(getOneStoryFrameItem("1"))
+        frames.add(getOneStoryFrameItem("2"))
+        frames.add(getOneStoryFrameItem("3"))
+
+        // When
+        saveStoryGutenbergBlockUseCase.saveNewLocalFilesToStoriesPrefsTempSlides(
+                siteModel,
+                0,
+                frames
+        )
+
+        // Then
+        verify(storiesPrefs, times(3)).saveSlideWithTempId(any(), any(), any())
+    }
+
+    private fun setupFluxCMediaFiles(
+        emptyList: Boolean
+    ): ArrayList<MediaFile> {
+        when (emptyList) {
+            true -> return ArrayList()
+            false -> {
+                val mediaFiles = ArrayList<MediaFile>()
+                for (i in 1..10) {
+                    val oneMediaFile = MediaFile()
+                    oneMediaFile.id = i
+                    oneMediaFile.mediaId = (i + 1000).toString()
+                    oneMediaFile.mimeType = "image/jpeg"
+                    oneMediaFile.fileURL = "https://testsite.files.wordpress.com/2020/10/wp-0000000.jpg"
+                    mediaFiles.add(oneMediaFile)
+                }
+                return mediaFiles
+            }
+        }
+    }
+
+    private fun getOneMediaFile(id: Int): MediaFile {
+        val oneMediaFile = MediaFile()
+        oneMediaFile.id = id
+        oneMediaFile.mediaId = (id + 1000).toString()
+        oneMediaFile.mimeType = "image/jpeg"
+        oneMediaFile.fileURL = "https://testsite.files.wordpress.com/2020/10/wp-0000000.jpg"
+        return oneMediaFile
+    }
+
+    private fun getOneStoryFrameItem(id: String): StoryFrameItem {
+        return StoryFrameItem(
+                source = mock(),
+                id = id
+        )
+    }
+
+    private fun setupMediaFileDataList(
+        emptyList: Boolean
+    ): ArrayList<StoryMediaFileData> {
+        when (emptyList) {
+            true -> return ArrayList()
+            false -> {
+                val mediaFiles = ArrayList<StoryMediaFileData>()
+                for (i in 1..10) {
+                    val oneMediaFile = StoryMediaFileData(
+                            id = i.toString(),
+                            mime = "image/jpeg",
+                            link = "https://testsite.files.wordpress.com/2020/10/wp-0000000.jpg",
+                            url = "https://testsite.files.wordpress.com/2020/10/wp-0000000.jpg",
+                            alt = "",
+                            type = "image",
+                            caption = ""
+                    )
+                    mediaFiles.add(oneMediaFile)
+                }
+                return mediaFiles
+            }
+        }
+    }
+
+    companion object {
```

Comment:
Could the constants in this companion object use the uppercase naming convention. When I was looking at their usages in the tests I ended up looking for a variable within the test itself. Thanks :) 

`blockWithEmptyMediaFiles` -> `BLOCK_WITH_EMPTY_MEDIA_FILES`

Reply:
😅 🤕 🤦 IDK how I missed that one 🤷‍♂️ - done in 0575171172

Reply:
😅